client/my-sites/site-settings/form-general.jsx

```diff
@@ -310,6 +310,73 @@ class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	netNeutralityOption() {
+		const {
+			fields,
+			isRequestingSettings,
+			translate,
+			handleToggle,
+			moment,
+			handleSubmitForm,
+			isSavingSettings,
+		} = this.props;
+
+		const today = moment(),
+			lastDay = moment( { year: 2017, month: 11, day: 14 } );
```

Comment:
See https://github.com/Automattic/wp-calypso/pull/16047/files#r126759726:

> As a note, days and years are 1-indexed, and other things are 0-indexed.

I.e. this is December (not November) 14.